cluster/hooks.go

```diff
@@ -533,6 +538,27 @@ func InstallIngressControllerPostHook(input interface{}) error {
 		return errors.Errorf("Get User failed : %s", err.Error())
 	}
 
+	orgID := cluster.GetOrganizationId()
+	organization, err := auth.GetOrganizationById(orgID)
+	if err != nil {
+		return emperror.WrapWith(err, "failed to get organization", "orgID", orgID)
+	}
+
+	domainWithOrgName := fmt.Sprintf("%s.%s", organization.Name, viper.GetString(pipConfig.DNSBaseDomain))
+
+	ds := domains{
+		Enabled: true,
+		DomainsList: []map[string]interface{}{
+			{
+				"main": fmt.Sprintf("*.%s", domainWithOrgName)},
+			{
+				"sans": []string{
+					fmt.Sprintf("%s.%s", cluster.GetName(), domainWithOrgName),
+					domainWithOrgName},
+			},
+		},
+	}
+
 	acme := map[string]interface{}{
```

Comment:
Sorry, I might not have been clear: the whole thing could be refactored into struct